src/units/unit.cpp

```diff
@@ -2106,6 +2106,14 @@ void unit::apply_builtin_effect(std::string apply_to, const config& effect)
 		set_attr_changed(UA_ATTACKS);
 		for(attack_ptr a : attacks_) {
 			a->apply_modification(effect);
+			for(const config& specials : effect.child_range("specials")) {
```

Comment:
It's called `[set_specials]` [here](https://wiki.wesnoth.org/EffectWML#apply_to-attack).

I kinda think it belongs in `attack::apply_modification`, but I don't think that's called anywhere else anyway, so maybe it's a little neater to handle it here.